benchmark/tracer.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
 const benchmark = require('./benchmark');
-const opentelemetry = require('@opentelemetry/core');
-const { BasicTracerRegistry, BatchSpanProcessor, InMemorySpanExporter, SimpleSpanProcessor } = require('@opentelemetry/tracing');
-const { NodeTracerRegistry } = require('@opentelemetry/node');
+const opentelemetry = require('../packages/opentelemetry-core');
+const { BasicTracerRegistry, BatchSpanProcessor, InMemorySpanExporter, SimpleSpanProcessor } = require('../packages/opentelemetry-tracing');
 
-const exporter = new InMemorySpanExporter();
 const logger = new opentelemetry.NoopLogger();
 
 const setups = [
```

Comment:
I think these setups make more sense since Basic and Node TracerRegistries return the same Tracer class so their performance should be identical.